From 57c1daa4566bf3eda877dc83f57def42e383e845 Mon Sep 17 00:00:00 2001
From: Wendy Liang <wendy.liang@xilinx.com>
Date: Wed, 16 Sep 2020 16:47:35 -0700
Subject: [PATCH 1597/1851] misc: xilinx-ai-engine: remove unnecessary print

commit 6d137f69222f167fa4bba35689f3a0ef9aab195b from
https://github.com/Xilinx/linux-xlnx.git

Remove unnecessary print when probe() fails to get irq to solve
warning:

drivers/misc/xilinx-ai-engine/ai-engine-dev.c:451:2-9: line 451 is
redundant because platform_get_irq() already prints an error

Reported-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Wendy Liang <wendy.liang@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/misc/xilinx-ai-engine/ai-engine-dev.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/misc/xilinx-ai-engine/ai-engine-dev.c b/drivers/misc/xilinx-ai-engine/ai-engine-dev.c
index cd88a1a5d27a..6afd0c6aecd0 100644
--- a/drivers/misc/xilinx-ai-engine/ai-engine-dev.c
+++ b/drivers/misc/xilinx-ai-engine/ai-engine-dev.c
@@ -447,10 +447,8 @@ static int xilinx_ai_engine_probe(struct platform_device *pdev)
 	INIT_WORK(&adev->backtrack, aie_array_backtrack);
 
 	adev->irq = platform_get_irq_byname(pdev, "interrupt1");
-	if (adev->irq < 0) {
-		dev_err(&pdev->dev, "Failed to find AIE interrupt1\n");
+	if (adev->irq < 0)
 		goto free_ida;
-	}
 
 	ret = devm_request_irq(dev, adev->irq, aie_interrupt, 0, dev_name(dev),
 			       adev);
-- 
2.31.1

